FrontEnd/src/colors.scss

```diff
@@ -2,6 +2,8 @@
     --gold-primary: #BB8F52;
     --gold-accent: rgba(201, 170, 136, 30);
 
+    --blue-primary: #D0DBEA;
```

Comment:
hmm, you've added this, but never used it?

Reply:
left over from testing i think